### ISSUE TITLE
Create docker-compose system test setup

### ISSUE THREAD

**Issue by frankscholten** [mesos/elasticsearch#65]
Let's setup a way to run automated system tests with docker-compose.

The steps are 

* docker-compose up - either via Gradle or within JUnit
* Perform assertions such as calling the Elasticsearch endpoints to check if node discovery works
* docker-compose rm

**Comment by frankscholten** [comment]
Once the setup is in place let's create separate issues to test each major scenario such as node discovery, slave lost, executor lost, the entire Scheduler and Executor API.

**Comment by frankscholten** [comment]
Merged into master
